tests/cfr/assets/sys_operations.sql

```diff
@@ -1,4 +1,4 @@
-CREATE TABLE IF NOT EXISTS "sys-operations" (
+CREATE TABLE IF NOT EXISTS "crate"."sys_operations" (
```

Comment:
The idea was that the user can specify the schema, for example like this:

```shell
export CRATEDB_SQLALCHEMY_URL=crate://localhost/?schema=foobar
```

-- https://cratedb-toolkit--88.org.readthedocs.build/cfr/#synopsis

Reply:
Yeah,  this was more tbd. I'm fine to drop this.

Reply:
I dropped this now. Should be done in a separate PR, but the effort to have a fallback schema may be a bit high. Not really sure if this is worth it.